xbmc/pvr/recordings/PVRRecording.h

```diff
@@ -82,8 +84,23 @@ namespace PVR
     std::string   m_strFanartPath;    /*!< fanart path */
     unsigned      m_iRecordingId;     /*!< id that won't change while xbmc is running */
 
+  private:
```

Comment:
Okay, I thought you meant that it would be more efficient.
You cannot create copies if the copy ctor and operator= are private and have a default implementation, like we do throughout the code, so I'm not sure what you're on about here. Just make the ctor public again so you don't need the friend class statement.